Code/GraphMol/ChemReactions/Wrap/rdChemReactions.cpp

```diff
@@ -470,6 +470,8 @@ of the replacements argument.",
   python::def("HasReactantTemplateSubstructMatch",RDKit::hasReactantTemplateSubstructMatch,
 		  (python::arg("reaction"), python::arg("queryReaction")),
           "tests if the reactants of a queryReaction are substructures of the reactants of a reaction");
-
+  python::def("UpdateProductsStereochemistry",RDKit::updateProductsStereochem,
+          (python::arg("reaction")),
+          "Updates the stereochemistry of the product by considering 4 different cases: inversion, retention, removal, and introduction");
```

Comment:
Just a thought:
I'm not sure how to phrase this exactly, but the documentation for this should probably include some bit of text to indicate that this is really an expert-user function that you shouldn't be calling if you don't know what it means. Perhaps something like: "This is done by default by the RXN and SMARTS parsers and should really only be called if reactions have been constructed some other way."?